### ISSUE TITLE
[BotW] Reflections are incorrect

### ISSUE THREAD

**Issue by alexkiri** [slashiee/cemu_graphic_packs#100]
Reflection, where present (at the moment I only know about shrine wall reflections), are incorrectly displayed.
This is the correct reflection, in 720p
![reflcorrect](https://user-images.githubusercontent.com/6061770/31316211-ba1d94d6-ac31-11e7-807f-62e110a3c439.png)
This is incorrect, in 1440p
![reflincorrect2k](https://user-images.githubusercontent.com/6061770/31316210-ba1d60a6-ac31-11e7-901a-e01c28dd758e.png)
The reflection appears squished horizontally. And in only appears at a much sharper angle. Also, there are also reflections on floors, those appear to be squished vertically. Those are more subtle, but you can see in the 720p screenshot that Link's boots can be seen reflected. Probably those are on 1440p screenshot, but squished vertically, so they are sort of indistinguishable.

**Comment by alexkiri** [comment]
In the pics above, you can look at the blue light runes. It's much more obvious to see how squished the reflection is.

**Comment by alexkiri** [comment]
![clipboard01](https://user-images.githubusercontent.com/6061770/31316319-b6c8ee8c-ac33-11e7-9d70-b1da9c638bb0.png)
My findings with nsight led me to this

**Comment by Crementif** [comment]
I don't know if that picture has anything to do with reflectings.

**Comment by alexkiri** [comment]
They are related. The final reflection pass uses these 2 textures. I'll have to dig deeper, but i belive that shader, or something related to it, is the cause

**Comment by Crementif** [comment]
From what I'm seeing you only 1 texture is used. uf_remappedPS[0] that is.

**Comment by Crementif** [comment]
You should check the bloom commit @NAVras-Z did which had stretching issues.

**Comment by alexkiri** [comment]
Yup. Those fixes from Navras are included.

**Comment by alexkiri** [comment]
I investigated this further, so I thought I should document this. I think I found the root cause.
The reflections are calculated in multiple passes, sequentially. I compared some render targets upscaled to 1440 (top) with the native 720p (bottom)
![normals](https://user-images.githubusercontent.com/6061770/31318549-20f6fb22-ac5d-11e7-83f8-f88099821924.png)
![clipped](https://user-images.githubusercontent.com/6061770/31318551-26a3e4f4-ac5d-11e7-9868-b2015790011c.png)
![reflects](https://user-images.githubusercontent.com/6061770/31318552-2809c246-ac5d-11e7-9c15-7b91179c0e91.png)
Somehow the slight variation in the first image has big impact on the reflections
